content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -11,73 +11,117 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 
 ## ASP.NET Core
 
-The following example demonstrates automatic and manual instrumentation via an
-ASP.NET Core app.
+The following example demonstrates library based and manual instrumentation via an ASP.NET Core app.
```

Comment:
"library based" is an undefined term.. Use "instrumentation library".

Reply:
Reworded with that in mind.